pkg/util/utils.go

```diff
@@ -557,6 +557,8 @@ func ValidatePullType(pullType string) (PullType, error) {
 		return PullImageAlways, nil
 	case "missing":
 		return PullImageMissing, nil
+	case "IfNotPresent":
```

Comment:
perhaps pullType should be forced to lower() ?

Reply:
Does Kube allow ifnotpresent?